### ISSUE TITLE
[Request]: `Run Executable` should default to the Bottle's directory

### ISSUE THREAD

**Issue by MasterKia** [bottlesdevs/Bottles#3121]
### Tell us the problem or your need

Currently the `Run Executable` button defaults to whatever the last directory has been, even on the **first** try after creating the Bottle.

### Describe the solution you'd like

The `Run Executable` should always default to that Bottle's directory e.g. `/home/[...]/.var/app/com.usebottles.bottles/data/bottles/bottles/[...]/drive_c/`

For instance, for games we sometimes copy mods to the game's folder and then need to run that mod and manually navigating to that path or even using the "Browser files" option it Bottles can be unintuitive. 

Hopefully this is supported by the Flatpak sandbox.

**Comment by HalJordan** [comment]
Yeah, the default location for anything regarding a particular bottle should start in that bottle's directory, or let us set a system default(such as a preferred download location for installer files).
